### DISCUSSION TITLE
docker-compose and non-standard MQTT ports?

### DISCUSSION THREAD

**Discussion by AnotherDaniel** [teslamate-org/teslamate#1168] [Q&A]
Hi all - first, many thanks to @adriankumpf for this most excellent piece of software, enjoying it every day!

My question: when using a docker-compose deployment, is there a way us have teslamate use non-standard ports for interacting with mosquitto? I see there's a MQTT_HOST environment variable, but nothing for declaring the port to use (or is _HOST working with a :<port> notation? Admit I haven't tried that yet).

**Comment by adriankumpf** [comment]
This is currently not yet possible. The ports are hard coded to 1883/8883.
